libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1671,8 +1716,16 @@ public void onClick(DialogInterface dialog, int id) {
                     return;
                 }
 
+                if (mediaType == MediaType.AUDIO) {
```

Comment:
Let's just remove this whole block, we don't want to open an "audio settings screen" here for now